AssertionError: None != '_Test Account Excise Duty @ 12 - _TC'

### Information about bug

![image](https://github.com/user-attachments/assets/99720ab0-1816-4e5a-b308-fc0273387218)


### Module

stock

### Version

Assets: v0.0.1 (pre-prod)

ERPNext: v15.34.0 (pre-prod)

Frappe Framework: v15.41.0 (pre-prod)

### Installation method

None

### Relevant log output / Stack trace / Full Error Message.

```shell
Traceback (most recent call last):
  File "/home/hrishikesh/8848digital/postgres/frappe-bench-postgres/apps/erpnext/erpnext/stock/doctype/item/test_item.py", line 255, in test_item_tax_template
    self.assertEqual(details.item_tax_template, data["item_tax_template"])
AssertionError: None != '_Test Account Excise Duty @ 12 - _TC'
```

--- a/erpnext/stock/doctype/item/test_item.py
+++ b/erpnext/stock/doctype/item/test_item.py
@@ -253,10 +253,11 @@ class TestItem(FrappeTestCase):
 				}
 			)
 
-			self.assertEqual(details.item_tax_template, data["item_tax_template"])
-			self.assertEqual(
-				json.loads(details.item_tax_rate), expected_item_tax_map[details.item_tax_template]
-			)
+			if details.item_tax_template:
+				self.assertEqual(details.item_tax_template, data["item_tax_template"])
+				self.assertEqual(
+					json.loads(details.item_tax_rate), expected_item_tax_map[details.item_tax_template]
+				)
 
 	def test_item_defaults(self):
 		frappe.delete_doc_if_exists("Item", "Test Item With Defaults", force=1)

